code/game/machinery/computer/store.dm

```diff
@@ -31,7 +31,7 @@
 	var/dat = {"
 <html>
 	<head>
-		<title>[command_name()] Merchandise</title>
+		<title>Nanotrasen Merchandise</title>
 		<style type="text/css">
```

Comment:
The two in this file were replaced with "Nanotrasen" rather than "NAS Trurl"?
I mean it probably works better like that, but these seem to be the only things changed to this.

Reply:
Intentional change. It makes more sense here, however in other cases (mainly command reports), NAS Trurl makes sense